### PR TITLE
DEV-864 change get experiment endpoint

### PR DESCRIPTION
**Pull request by Cassiepang**

### CODE REVIEW

File: service-realization/src/main/java/sg/ncl/service/realization/logic/RealizationServiceImpl.java [hunk 1]
```diff
@@ -8,13 +8,17 @@
...
 import sg.ncl.service.realization.data.jpa.RealizationRepository;
+import sg.ncl.service.realization.domain.Realization;
 import sg.ncl.service.realization.domain.RealizationService;
 import sg.ncl.service.realization.domain.RealizationState;
 
 import javax.inject.Inject;
 import javax.validation.constraints.NotNull;
 import java.time.ZonedDateTime;
 import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
```
Comment (ChrisZhong) [inline service-realization/src/main/java/sg/ncl/service/realization/logic/RealizationServiceImpl.java]: MINOR Remove this unused import 'java.util.ArrayList'. [rule](https://sonarqube.com/coding_rules#rule_key=squid%3AUselessImportCheck)

File: service-realization/src/main/java/sg/ncl/service/realization/logic/RealizationServiceImpl.java [hunk 2]
```diff
@@ -33,6 +37,11 @@
     }
 
     @Transactional
+    public List<Realization> getAll() {
```
Comment (ChrisZhong) [inline service-realization/src/main/java/sg/ncl/service/realization/logic/RealizationServiceImpl.java]: MAJOR Add the "@Override" annotation above this method signature [rule](https://sonarqube.com/coding_rules#rule_key=squid%3AS1161)

**Comment by ChrisZhong** [comment]
SonarQube analysis reported 2 issues
* MAJOR 1 major
* MINOR 1 minor

Watch the comments in this conversation to review them.

File: service-realization/src/main/java/sg/ncl/service/realization/logic/RealizationServiceImpl.java [hunk 3]
```diff
@@ -33,6 +36,12 @@
     }
 
     @Transactional
```
Comment (ZHANGChunwang) [inline service-realization/src/main/java/sg/ncl/service/realization/logic/RealizationServiceImpl.java]: For get operation, you do not need @Transactional

File: service-realization/src/main/java/sg/ncl/service/realization/logic/RealizationServiceImpl.java [hunk 4]
```diff
@@ -33,6 +36,12 @@
...
+    @Override
+    public List<Realization> getAll() {
+        return realizationRepository.findAll().stream().collect(Collectors.toList());
+    }
+
+    @Transactional
```
Comment (ZHANGChunwang) [inline service-realization/src/main/java/sg/ncl/service/realization/logic/RealizationServiceImpl.java]: Remove @Transactional

**Pull request merged**